deployments/local/ops/readme.md

```diff
@@ -0,0 +1,53 @@
+Operations
+-----
+This shows how to work as operator with the current start.java --http setup
+
+#Monitoring and logging
+Monitoring and logging with [Grafana](#grafana).
+
+Background: 
+* Scraping metrics by [Prometheus](#prometheus).
+* Log aggregation with [Promtail](#promtail) und [Loki](#loki).
+
+Https is not tested so far.
+
+## Prometheus
+Keycloak does not protect [Prometheus](https://prometheus.io/).
```

Comment:
We cloud add an reverse proxy in front of those unprotected services to protect them

Reply:
In general we could decide if we want these services to be exposed and protected?
If they should be protected, I would work on the keycloak integration. Until then the ports are not exposed via docker by default? Is the proxy a proposal to use keycloak for protecting these sites?